app/src/main/java/com/ae/apps/tripmeter/fragments/TripsListFragment.java

```diff
@@ -85,6 +95,47 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         // Removing default profile feature
         // checkForDefaultProfile();
 
+        view=createViewHandlingPermissions(view,list);
```

Comment:
avoid assignments to variables that are passed in as parameters. Sonar violation